tests/compiler/test_basics.py

```diff
@@ -474,18 +474,22 @@ def get_one() -> "int":
     )
     with warnings.catch_warnings(record=True) as w:
         compiler.do_compile()
-        assert len(w) == 1 if decorator else len(w) == 0
-        if len(w):
-            warning = w[0]
-            assert issubclass(warning.category, PluginDeprecationWarning)
-            if replaced_by:
-                replaced_by_name = replaced_by.replace('"', "")
-                assert (
-                    f"Plugin 'get_one' in module 'inmanta_plugins.test_module' is deprecated. It should be "
-                    f"replaced by '{replaced_by_name}'" in str(warning.message)
-                )
-            else:
-                assert "Plugin 'get_one' in module 'inmanta_plugins.test_module' is deprecated." in str(warning.message)
+        if decorator:
+            has_warning: bool = False
+            for warning in w:
+                if issubclass(warning.category, PluginDeprecationWarning):
+                    has_warning = True
+                    if replaced_by:
+                        replaced_by_name = replaced_by.replace('"', "")
```

Comment:
Is there a specific reason why the `replaced_by` contains double quotes?

Reply:
I think this is a residue of multiple changes, I removed them :p